raven_2arm:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Position Controllers ---------------------------------------
  shoulder_L_position_controller:
    type: effort_controllers/JointPositionController
    joint: shoulder_L
    pid: {p: 30.0, i: 0.5, d: 0.08}
  elbow_L_position_controller:
    type: effort_controllers/JointPositionController
    joint: elbow_L
    pid: {p: 30.0, i: 0.5, d: 0.08}
  insertion_L_position_controller:
    type: effort_controllers/JointPositionController
    joint: insertion_L
    pid: {p: 100, i: 0.5, d: 0.12}
  tool_roll_L_position_controller:
    type: effort_controllers/JointPositionController
    joint: tool_roll_L
    pid: {p: 0.9, i: 0.0, d: 0.01}
  wrist_joint_L_position_controller:
    type: effort_controllers/JointPositionController
    joint: wrist_joint_L
    pid: {p: 0.5, i: 0.0, d: 0.0}
  grasper_joint_1_L_position_controller:
    type: effort_controllers/JointPositionController
    joint: grasper_joint_1_L
    pid: {p: 3, i: 0.0, d: 0.0}
  grasper_joint_2_L_position_controller:
    type: effort_controllers/JointPositionController
    joint: grasper_joint_2_L
    pid: {p: 3, i: 0.0, d: 0.0}

  

  shoulder_R_position_controller:
    type: effort_controllers/JointPositionController
    joint: shoulder_R
    pid: {p: 10, i: 0.2, d: 0.08}
  elbow_R_position_controller:
    type: effort_controllers/JointPositionController
    joint: elbow_R
    pid: {p: 10, i: 0.2, d: 0.08}
  insertion_R_position_controller:
    type: effort_controllers/JointPositionController
    joint: insertion_R
    pid: {p: 100, i: 0.2, d: 0.12}
  tool_roll_R_position_controller:
    type: effort_controllers/JointPositionController
    joint: tool_roll_R
    pid: {p: 0.9, i: 0.0, d: 0.01}
  wrist_joint_R_position_controller:
    type: effort_controllers/JointPositionController
    joint: wrist_joint_R
    pid: {p: 0.5, i: 0.0, d: 0.0}
  grasper_joint_1_R_position_controller:
    type: effort_controllers/JointPositionController
    joint: grasper_joint_1_R
    pid: {p: 10, i: 0.0, d: 0.0}
  grasper_joint_2_R_position_controller:
    type: effort_controllers/JointPositionController
    joint: grasper_joint_2_R
    pid: {p: 10, i: 0.0, d: 0.0}
